### PR TITLE
Add strawman national schema

### PR DESCRIPTION
**Pull request by benweissmann**

### CODE REVIEW

File: enip_backend/export/schemas/national.example.json [hunk 1]
```diff
@@ -0,0 +1,443 @@
...
+      {
+        "won": 22,
+        "party": "Dem"
+      },
+      {
+        "won": 11,
+        "party": "GOP"
+      },
+      {
+        "won": 2,
```
Comment (jasonkb) [inline enip_backend/export/schemas/national.example.json]: on the back-end I think we'll want to add this in to the Dem total

File: enip_backend/export/schemas/national.example.json [hunk 2]
```diff
@@ -0,0 +1,443 @@
...
+        "winner": false,
+        "popVote": 4489221,
+        "popPct": 0.03,
+        "statesWon": 0,
+        "electWon": 0
+      }
+    ],
+    "S": [
+      {
+        "won": 22,
```
Comment (jasonkb) [inline enip_backend/export/schemas/national.example.json]: I think we'll want to put in a key with the number of holdover seats for each party

File: enip_backend/export/schemas/national.example.json [hunk 3]
```diff
@@ -0,0 +1,443 @@
...
+        "won": 235,
+        "party": "Dem"
+      },
+      {
+        "won": 199,
+        "party": "GOP"
+      },
+      {
+        "won": 0,
+        "party": "Others"
```
Comment (jasonkb) [inline enip_backend/export/schemas/national.example.json]: I think there are no "Others" who can win a House seat?

File: enip_backend/export/schemas/national.example.json (hunk 2 above)
Comment (benweissmann) [reply enip_backend/export/schemas/national.example.json]: going to just hardcode this in the frontend

File: enip_backend/export/schemas/national.example.json (hunk 1 above)
Comment (benweissmann) [reply enip_backend/export/schemas/national.example.json]: Not going to worry about this because the holdovers are hardcoded

Comment (jasonkb) [reply enip_backend/export/schemas/national.example.json]: this is a moot point now that Bernie and Angus are part of holdovers, and we'll treat Al Gross as Dem

File: enip_backend/export/schemas/national.example.json (hunk 2 above)
Comment (jasonkb) [reply enip_backend/export/schemas/national.example.json]: front-end can deal with holdover math

**Review by itsjoekent — APPROVED** [review]
looks good to me!

**Review by jasonkb — APPROVED** [review]
Looks PERFECT! ahh the json schema will be such a good peace of mind.

**Pull request merged**